pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2016 The Kubernetes Authors.
```

Comment:
I'd suspect to reflect the initial time this was committed. We never update those.